nylas/models/availability.py

```diff
@@ -74,7 +74,7 @@ class OpenHours(TypedDict):
     timezone: str
     start: str
     end: str
-    extdates: List[str]
+    exdates: NotRequired[List[str]]
```

Comment:
https://nylas.slack.com/archives/C055R9J2E91/p1692121220250119?thread_ts=1692120688.645429&cid=C055R9J2E91

Changed the name to exdates